src/main/java/io/specto/hoverfly/junit/dsl/RequestedServiceBuilder.java

```diff
@@ -0,0 +1,21 @@
+package io.specto.hoverfly.junit.dsl;
+
+import io.specto.hoverfly.junit.dsl.matchers.PlainTextFieldMatcher;
+
+public class RequestedServiceBuilder extends AbstractServiceBuilder {
```

Comment:
It is slightly different in Hoverfly-Java,
- when simulating: 
```
service(matches("api*.flight.com"))
                    .get("/api/bookings")
                    .queryParam("airline", contains("Pacific"))
```
has body matcher set to `exactMatch: ""`, so that if the code accidentally call GET request with a body, the matching should fail. In other word, we want it to be a bit strict when simulating. 

when verifying:
```
requestedForService(matches("*.flight.*")).get("/api/bookings")
```
doesn't set body matcher, as we assume the users are only interested in searching for requests that match what they want.

When simulating, it is searching request/response mappings, while verifying is searching the journal.